IdentityCore/src/parameters/MSIDRequestParameters.h

```diff
@@ -59,6 +59,7 @@
 @property (nonatomic) NSUInteger tokenExpirationBuffer;
 @property (nonatomic) BOOL extendedLifetimeEnabled;
 @property (nonatomic) BOOL instanceAware;
+@property (nonatomic) BOOL allowGettingAccessTokenWithRefreshToken;
```

Comment:
The name is confusing. I added a similar comment in the MSAL repo.

Reply:
Renamed